maven/bnd-reporter-maven-plugin/src/test/resources/integration-test/test/report-multi-project/projectA/pom.xml

```diff
@@ -0,0 +1,54 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>biz.aQute.bnd-test</groupId>
+		<artifactId>report-multi-project</artifactId>
+		<version>0.0.1</version>
+	</parent>
+
+	<artifactId>projectA</artifactId>
+	<version>0.0.1</version>
+	<packaging>pom</packaging>
```

Comment:
I suspect this should really be `jar` packaging.

Reply:
Indeed! This is why my bundles wasn't built, thanks.